package.json

```diff
@@ -21,7 +21,7 @@
     "node-statsd": "0.0.7",
     "noxmox": "0.2.4",
     "nunjucks": "0.1.10",
-    "sequelize": "1.6.0",
+    "sequelize": "^3.8.0",
```

Comment:
updates here get weeded in subsequent PRs, they're mostly just to not break `npm install` in this PR